overlord/snapstate/snapmgr.go

```diff
@@ -59,6 +60,10 @@ func (ss *SnapSetup) MountDir() string {
 	return snap.MountDir(ss.Name, ss.Revision)
 }
 
+func (ss *SnapSetup) DevMode() bool {
+	return ss.Flags&int(snappy.DeveloperMode) != 0
```

Comment:
How is the whole thing working considering that snappy.DeveloperMode doesn't match snapstate.DevMode?